prow/cmd/hmac/fakeghhook/fakeghhook.go

```diff
@@ -123,6 +126,52 @@ func (f *FakeClient) DeleteRepoHook(org, repo string, id int, req github.HookReq
 	return deleteHook(f.RepoHooks, org+"/"+repo, req)
 }
 
+func (f *FakeClient) ListCurrentUserRepoInvitations() ([]github.UserRepoInvitation, error) {
+	return f.UserRepoInvitations, f.getNextError()
+}
+
+func (f *FakeClient) AcceptUserRepoInvitation(invitationID int) error {
+	err := f.getNextError()
+	if err != nil {
+		return err
+	}
+	// Must exist
+	found := -1
+	for idx, iv := range f.UserRepoInvitations {
+		if iv.InvitationID == invitationID {
+			found = idx
+			break
+		}
+	}
+	if found != -1 {
+		f.UserRepoInvitations = append(f.UserRepoInvitations[:found], f.UserRepoInvitations[found+1:]...)
```

Comment:
this can panic or not? If we have exactly one, we will end up calling `f.UserRepoInvitations[1:]...` on an one-element slice, right?